infra/build/functions/sync/deploy.sh

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+JOB_TOPIC=schedule-project-sync
+SCHEDULER_JOB=sync-scheduler
+JOB_SCHEDULE="*/30 * * * *"
+MESSAGE="Start Sync"
+ENTRY_POINT=sync
+
+if [ "$1" ]; then
+	gcloud config set project $1
```

Comment:
This sets the project globally for the user, which is not desired. Instead, please pass --project to all gcloud commands.